### PR TITLE
add vertices hover message 💭

### PR DESCRIPTION
**Pull request by victoriajuan**

### CODE REVIEW

File: src/RadarChart.js [hunk 1]
```diff
@@ -316,17 +248,23 @@ var radarChart = function () {
...
                                 .attr("cy", function(d, i) {
-                                    // console.log(d.coordinates.y);
                                     return d.coordinates.y;
                                 })
                                 .attr("fill", color(gColor))
                                 .attr("fill-opacity", 1)
-                                .on(over, showTooltip)
-                                .on(out, verticesTooltipHide);
+                                .on(over, function(d) {
+                                    verticesTooltip.style("opacity", 0.9)
```
Comment (zmbc) [inline src/RadarChart.js:259]: I'm slightly concerned that doing this with opacity will leave a "ghost" element covering something up. Does it work if you do `visibility: hidden` and `visibility: visible` instead? That's what we use for the tooltips on other charts.

File: src/RadarScatter.js [hunk 2]
```diff
@@ -266,7 +265,6 @@ var RadarScatter = function() {
                 // Customize the chart for being tiny!
                 .lineThickness(0.5)
                 .showVertice(false)
-                .showTooltip(false)
```
Comment (zmbc) [inline src/RadarScatter.js:269]: If there's no `.showTooltip(false)` here, are there tooltips showing up on the tiny versions?

**Review by zmbc — CHANGES_REQUESTED** [review]
This looks great! Just two questions:

File: src/RadarScatter.js (hunk 2 above)
Comment (victoriajuan) [reply src/RadarScatter.js:269]: So if you look at the vertice part, I will only show hover message when showVertiece is true which is in the large-radar div. Does it make sense?

File: src/RadarChart.js (hunk 1 above)
Comment (victoriajuan) [reply src/RadarChart.js:259]: so I change to this and it won't work, not sure if i miss write something?
verticesTooltip
                                        .html("<strong>Race</strong>: " + d.axis + "<br />" +
                                        "<strong>Percentile</strong>: " + d.value.toFixed(2) + "<br />")
                                        .style('visibility', 'visible')
                                        .style("left", (d3.event.pageX) + "px")
                                        .style("top", (d3.event.pageY) + "px");

File: src/RadarScatter.js (hunk 2 above)
Comment (zmbc) [reply src/RadarScatter.js:269]: :+1:

File: src/RadarChart.js (hunk 1 above)
Comment (zmbc) [reply src/RadarChart.js:259]: Hmm, don't immediately know what's up and it's not a big deal. Merging.

**Review by zmbc — APPROVED** [review]

**Pull request merged**